Microsoft.Research/ClousotMain/ClousotMain.cs

```diff
@@ -795,6 +796,10 @@ private bool FilterPreconditionsAccordingToWarningLevel(BoxedExpression exp)
 
       public int Analyze()
       {
+        // String formatting in the analyzer does not specify a culture, therefore some tests fail on systems with non-English culture with e.g. "expected: 2.0, actual: 2,0"
+        // Setting the threads culture should be removed as soon as issue #149 is fixed.
+        Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
```

Comment:
> The tests are just calling the .exe - we would need another argument to pass the culture to the other process.

That would be preferable. My vote is for [`/LCID` to match **devenv.exe**](https://msdn.microsoft.com/en-us/library/9cytz106.aspx).